code/__DEFINES/lights.dm

```diff
@@ -1,5 +1,5 @@
 ///How much power emergency lights will consume per tick
-#define LIGHT_EMERGENCY_POWER_USE (0.1 KILO WATTS)
+#define LIGHT_EMERGENCY_POWER_USE (0.0001 * STANDARD_CELL_RATE)
```

Comment:
i feel like the defines are just making things harder to read

Reply:
True, but we need standards and  not magic numbers.  We have material sheet defines for mats and standard cell defines for power 

Also Hi jlsnow :) hope your doing well